pandas/tests/io/parser/test_network.py

```diff
@@ -56,50 +56,62 @@ def tips_df(datapath):
 @td.skip_if_not_us_locale()
 class TestS3:
     @td.skip_if_no("s3fs")
-    def test_parse_public_s3_bucket(self, tips_df):
+    def test_parse_public_s3_bucket(self, tips_df, s3so):
```

Comment:
Why does this test (and similar ones) need to change? I understand the changes to the moto-based ones, but not these which (I think) aren't using moto.

Reply:
`@pytest.mark.usefixtures("s3_resource")` a few lines above was installing the mocks in the previous version. I don't know what the intent was, but these tests never touched real S3 (and there is no public "pandas-test" bucket).

Reply:
Ahh gotcha. Thanks!